CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 # Changelog
+Attention, this release contains critical security fixes. Please update as soon as possible. Additionally, we are
+encouraging users to change your Sunshine password, especially if you expose the web UI (i.e. port 47790 by default)
+to the internet, or have ever uploaded your logs with verbose output to a public resource.
+
+## [0.18.0] - 2023-01-25
+### Added
```

Comment:
HDR is in :)

Reply:
We can also link to the new docs, since users will probably have a bunch of questions (if the testing is anything to go by).

https://docs.lizardbyte.dev/projects/sunshine/en/nightly/about/usage.html#hdr-support (that's a nightly link, but it should go to the regular link when the release is published).